server/src/main/java/org/opensearch/cluster/metadata/AliasMetadata.java

```diff
@@ -129,6 +129,13 @@ public CompressedXContent getFilter() {
         return filter();
     }
 
+    public String getFilterAsString() {
```

Comment:
I am wondering it adding this method is really needed (the `filter()` is exposed already), the consumer could do that transformation easily